src/components/SnapshotDetails/tabs/__tests__/SnapshotPipelineRunsList.spec.tsx

```diff
@@ -192,7 +192,7 @@ describe('SnapshotPipelinerunsTab', () => {
     const button = screen.queryByText('Add component');
     expect(button).toBeInTheDocument();
     expect(button.closest('a').href).toContain(
-      `http://localhost/workspaces/test-ns/import?application=my-test-app`,
+      `http://localhost/ns/test-ns/import?application=my-test-app`,
```

Comment:
We changed a lot from 'workspace' to 'ns' in our tests scripts. How about we enjoy NAMESPACE_LIST_PATH in link test like `${NAMESPACE_LIST_PATH}/test-ns/import?application=my-test-app`? 
However, we do not have to do it for i guess we do not need to change the ns to others. :-).

Reply:
Yes, We can use the same utilties for unit tests also but let's not migrate all of the unit tests, any new tests or modified tests can start to use those utilities as needed